src/main/telemetry/jetiexbus.c

```diff
@@ -116,7 +123,7 @@ typedef struct exBusSensor_s {
 // list of telemetry messages
 // after every 15 sensors a new header has to be inserted (e.g. "BF D2")
 const exBusSensor_t jetiExSensors[] = {
-    {"BF D1",           "",         EX_TYPE_DES,   0              },     // device descripton
+    {"Rotorflight D1",           "",         EX_TYPE_DES,   0              },     // device descripton
```

Comment:
Please maintain the same indentation.
One intend = 4 spaces. NO TABS!

Reply:
I suspect this was there already in the code as a tab as only thing bg changed was the bf to rotorflight.

Will adjust